src/sentry/static/sentry/app/views/settings/account/apiTokens.tsx

```diff
@@ -107,12 +107,6 @@ export class ApiTokens extends AsyncView<Props, State> {
             }
           )}
         </TextBlock>
-        <TextBlock>
-          <small>
-            psst. Looking for the <strong>DSN</strong> for an SDK? You'll find that under{' '}
```

Comment:
are you sure about removing this one?

Reply:
yeah, we got feedback that having these long term "Looking for..?" messages all over the place is not a good way to educate users where they should find stuff

the one from projectGeneralSettings will be removed in a few months